gluten-ut/spark35/src/test/scala/org/apache/spark/sql/GlutenDateFunctionsSuite.scala

```diff
@@ -29,6 +29,34 @@ class GlutenDateFunctionsSuite extends DateFunctionsSuite with GlutenSQLTestsTra
 
   private def secs(millis: Long): Long = TimeUnit.MILLISECONDS.toSeconds(millis)
 
+  test("unix_timestamp/to_unix_timestamp") {
```

Comment:
Seems there is no need to add a new test. We can just remove the below commented code to allow test legacy policy.

https://github.com/apache/incubator-gluten/blob/main/gluten-ut/spark35/src/test/scala/org/apache/spark/sql/GlutenDateFunctionsSuite.scala#L118

https://github.com/apache/incubator-gluten/blob/main/gluten-ut/spark35/src/test/scala/org/apache/spark/sql/GlutenDateFunctionsSuite.scala#L237

Ditto for spark32/33/34.